static/documents/pdt-v2-art-healthcert.json

```diff
@@ -1,335 +1,334 @@
-          }
+            }
+        ],
+        "$template": {
+            "name": "784d5e0e-054e-4199-82cc-eb44fa40dd0a:string:HEALTH_CERT",
+            "type": "66ffb36c-6929-435d-82cd-04d710e9f316:string:EMBEDDED_RENDERER",
+            "url": "52334e20-1e6f-4061-b190-38fa86b96d3e:string:https://healthcert.renderer.moh.gov.sg/"
         },
-        {
-          "fullUrl": "60513823-0f57-44cb-a527-5f146482d4b5:string:urn:uuid:fa2328af-4882-4eaa-8c28-66dab46950f1",
-          "resource": {
-            "resourceType": "4aece6f4-0004-43ae-b54b-c7d11bc46ef6:string:Organization",
-            "name": "479de6d8-4c80-4811-9c62-877c2991c769:string:MacRitchie Medical Clinic",
-            "type": [
-              {
-                "coding": [
-                  {
-                    "system": "2c9dd559-89b4-453c-8199-46c3d2bcae26:string:http://terminology.hl7.org/CodeSystem/organization-type",
-                    "code": "d966819b-c21d-432b-bd4d-27b4e340623b:string:prov",
-                    "display": "0b129b9e-b370-40a2-866a-bec312131b2a:string:Healthcare Provider"
-                  }
-                ],
-                "text": "17b72f2f-d8ce-42ea-a50e-f690faaa515f:string:Licensed Healthcare Provider"
-              }
-            ],
-            "contact": [
-              {
-                "telecom": [
-                  {
-                    "system": "b1879b15-377c-4234-b3c2-50ef571c895c:string:url",
-                    "value": "bb0a3e66-32b2-4e18-88bb-4f801872587e:string:https://www.macritchieclinic.com.sg"
-                  },
-                  {
-                    "system": "d1c2c1d5-01aa-4255-859f-e320b7b60ac9:string:phone",
-                    "value": "5cfea331-920f-4944-a113-a6995f47720a:string:+6561234567"
-                  }
-                ],
-                "address": {
-                  "type": "0ce52927-62f7-4836-ac9a-4d8bff374740:string:physical",
-                  "use": "01540af1-e1f1-4a62-8947-0c0e3518bcc1:string:work",
-                  "text": "287bf3a2-b2cc-4802-9c33-e601f692ae36:string:MacRitchie Hospital, Thomson Road, Singapore 123000"
-                }
-              }
-            ]
-          }
+        "notarisationMetadata": {
+            "reference": "9d22aacf-c3a5-492f-b717-a230c631e467:string:78073ba2-2a4a-4e62-9b09-a537d3c93f34",
+            "notarisedOn": "de3b68ff-c29a-44f9-845f-9da2b1b57e60:string:2022-03-11T01:58:09.723Z",
+            "passportNumber": "18539051-9b23-457a-ad48-e4dd0d544263:string:E7831177G",
+            "url": "21647938-88b4-4ec5-9119-43efd801414d:string:https://dev.verify.gov.sg/verify?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22http%3A%2F%2Flocalhost%3A5000%2Fstg%2Fdocument%2Fec4bcba7-5115-4349-af23-8fac4f53a9af%22%2C%22permittedActions%22%3A%5B%22VIEW%22%2C%22STORE%22%5D%2C%22redirect%22%3A%22https%3A%2F%2Fdev.verify.gov.sg%2Fverify%22%7D%7D#%7B%22key%22%3A%220304316b78888aa483fa1a34bdf445cf7dcbbf36eb8452194adf89b798bee6ad%22%7D"
```

Comment:
Hey @eugbyte, I think this URL is not valid.

Perhaps I can help you update the ART sample in #77 as well?

Reply:
Hey thanks!

Reply:
so i close this PR?

Reply:
Sure. Can help me approve the other PR too 🙏🏻